packages/embark/src/lib/core/logger.js

```diff
@@ -84,7 +83,7 @@ Logger.prototype.registerAPICall = function (plugins) {
     '/embark-api/logs',
     (ws, _req) => {
       self.events.on("log", function (logLevel, logMsg) {
-        logMsg = escapeHtml(logMsg);
+        logMsg = util.escapeHtml(logMsg);
```

Comment:
This doesn't look right to me. `escapeHtml()` originated from `utils/escapeHtml`. This probably needs to be changed back to `escapeHtml()` with `import { escapeHtml } from 'embark-utils'`